doc/source/data/examples/BUILD

```diff
@@ -5,3 +5,18 @@ filegroup(
     srcs = glob(["*.ipynb"]),
     visibility = ["//doc:__subpackages__"]
 )
+
+# --------------------------------------------------------------------
+# Test all doc/source/data/examples notebooks.
+# --------------------------------------------------------------------
+
+# big_data_ingestion.ipynb is not tested right now due to large resource requirements
+# and a need of a general overhaul.
+
+py_test_run_all_notebooks(
+    size = "large",
+    include = ["*.ipynb"],
+    exclude = [],
+    data = ["//doc/source/data/examples:data_examples"],
+    tags = ["exclusive", "team:ml"]
```

Comment:
should be "team:data"? As it's in the "data" directory.
Also, need to add the "gpu" tag.

Reply:
by the way, I just merged #35143. You'll need to resolve the conflicts.